rpm/update-spec-provides.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# This script will update the Version field in the spec which is set to 0 by
+# default. Useful for local manual rpm builds where the Version needs to be set
+# correctly.
```

Comment:
Comment is copypasted from the next script, and is misleading. And actually, since this script isn't even used, I might suggest removing it and adding it back when you fix the TODO. I won't block over that though.

Reply:
err, yes I need to change that comment. I'm actually also checking with the packit people on the side.

Reply:
updated comment.